wscript_build.py

```diff
@@ -139,6 +139,25 @@ def build(ctx):
     ctx(features = "ebml_header", target = "ebml_types.h")
     ctx(features = "ebml_definitions", target = "ebml_defs.c")
 
+    def _run(cmd):
```

Comment:
Is it possible to move some or all of this to the `waftools/checks/custom.py` so it's done in the configure stage.